config/configmaps/install-log-regexes-configmap.yaml

```
@@ -93,6 +93,11 @@ data:
       - "Quota \'SSD_TOTAL_GB\' exceeded"
       installFailingReason: GCPQuotaSSDTotalGBExceeded
       installFailingMessage: GCP quota SSD_TOTAL_GB exceeded
+    - name: GCPComputeQuota
+      searchRegexStrings:
+      - "compute.googleapis.com/cpus is not available in [a-z0-9-]* because the required number of resources"
```

I was just reviewing the previous iteration when you uploaded this. The notes still apply, especially if you want to keep the full regex you had before:

Interpreting a regex string through yaml is icky. This needs to be:
```suggestion
      - "compute\\.googleapis\\.com/cpus is not available in [a-z0-9-]* because the required number of resources \\([0-9]*\\) is more than remaining quota"
```
- Double backslash gets interpreted first by yaml into single backslash, which is what you want as an escape in the actual regex to remove special meaning from the subsequent character.
- Escape the periods in `compute.googleapis.com`; otherwise they'll match any single character. (Including period, so it'd still work, but your intent is to match exactly period.)
- Slash isn't special, so it doesn't need to be escaped. (You're probably thinking of needing to escape slash in a `sed` expression that's using slash as the regex delimiter. But e.g. `s/foo\/bar/foo\/baz/` is the same as `s,foo/bar,foo/baz,`. In this case you can think of double-quote as the delimiter (sort of).)
- Escaping to match literal parens is indeed necessary; but the backslashes need to be doubled as noted above.

---

oh, dang, thanks! i was trying to figure out why this regex wasn't matching even after escaping the parens (once)